Sources/CollectionViewModel.swift

```diff
@@ -167,3 +167,22 @@ public struct CollectionSectionViewModel: DiffableViewModel {
         self.diffingKey = diffingKey
     }
 }
+
+/// Collection support for diffing
+extension CollectionSectionViewModel: Collection {
```

Comment:
nit: maybe say "`CollectionSectionViewModel` conforms to `Collection`" or leave as `/// :nodoc:`

Reply:
Why is `CollectionSectionViewModel conforms to Collection` more informative than `CollectionSectionViewModel: Collection` + the current comment?

Reply:
Sorry just feel like that comment is a bit redundant. Just in case, I add backticks around `Collection`, which might make it read a bit better.